svg/elements/feComposite.json

```diff
@@ -332,44 +356,43 @@
         },
         "lighterForError": {
```

Comment:
This is not your fault, but I wouldn't mind if we fixed this in the process of doing all this work here:

`lighterForError` should be a subfeature `lighter_value` (description: `<code>lighter<code> value`) of the `operator` feature. We could work through all the other changes that need to happen, then, as a final polish to this PR, move the feature underneath `operator`.